### PR TITLE
stage 2 of gbq refactor

### PR DESCRIPTION
**Pull request by elray1**
This PR includes the following cleanup:

 - removes unused and duplicated imports
 - adds config files specifying model settings
     - only one of these settings is different for the `gbq_qr` and `gbq_qr_no_level` models, but the plan is to add more model variations changing other settings, and more settings that can be changed.
 - add argument parsing, with arguments to specify the reference date for the forecast (not really used, currently), the model name (used in file names for outputs, and to look up the config file to use for model settings), and a flag to do a short run for testing purposes (or if not set, a full run for producing an actual submission file)
 - put all the rest of the code into a `main()` function.  This code all still needs to be refactored, to be done in a future PR.

It's easier to view this one commit at a time, but unfortunately git's diff is not producing useful results for the changes made in the third commit, so it's a bit hard to parse.  A lot of what happened is that code got indented 2 spaces.

With your working directory in `code/gbq`, you should be able to do a test run with the following (potentially after rebuilding the flusion conda environment if you haven't looked at this repo recently -- see readme in base of repo)

```
conda activate flusion
python gbq_qr.py --short_run
```
This will produce a csv file under `submissions-hub/model-output/UMass-gbq_qr`.

### CODE REVIEW

**Comment by matthewcornell** [comment]
Per our conversation today:

# testing:
- recommend adding tests, from unit-level on up. perhaps a good starting point is incorporating the above app `python gbq_qr.py --short_run` as an integration-level test where the output .csv file is compared to an expected output file, modulo file name

# coding conventions:
- recommend following https://peps.python.org/pep-0008/#indentation indentation (_Use 4 spaces per indentation level._): 2 spaces is used in this project, but recommend 4 across all reichlab Python projects

# small things:
- unused imports: gbq_qr.py: `import math`
- statement seems to have no effect: gbq_qr.py: `test_pred_qs.shape`
- Unused local symbols: gbq_qr.py: `oob_pred_qs = np.nanmedian(oob_preds_by_bag, axis=1)`

**Comment by elray1** [comment]
Thanks. I've addressed all of these points other than the most important one (testing).

I propose to merge this PR and handle that piece in the next step, as some additional setup is required to enable reproducible testing.
